content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 10
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks (Galley and Sidecar Injector). In current Istio implementation,
+Galley and Sidecar Injector manage their own webhook configurations, which from
+security perspective is not recommended because a compromised webhook may conduct
+privilege escalation attacks.
+
+This task shows how to use `istioctl`, instead of Galley and Sidecar Injector, to
+manage the webhook configurations of Galley and Sidecar Injector.
+
+## Before you begin
+
+* Create a Kubernetes cluster with Istio installed. In the installation,
+[`global.operatorManageWebhooks`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) should
+be set as true, and [DNS certificates should be configured](/docs/tasks/security/dns-cert).
+Istio installation guides can be found [here](/docs/setup/install).
+
+* Install [`jq`](https://stedolan.github.io/jq/) for json parsing.
+
+## Check webhook certificates
+
+The webhook certificates generated are stored in the secrets specified in the configuration.
+
+1.  Check that webhook certificates have been generated:
+
+    {{< text bash >}}
+    $ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+    $ kubectl get secret dns.istio-sidecar-injector-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+    {{< /text >}}
+
+    The output from the above commands should include the DNS names of Galley and Sidecar Injector, respectively:
+
+    {{< text plain >}}
+    X509v3 Subject Alternative Name:
+      DNS:istio-galley.istio-system.svc, DNS:istio-galley.istio-system
+    {{< /text >}}
+
+    {{< text plain >}}
+    X509v3 Subject Alternative Name:
+      DNS:istio-sidecar-injector.istio-system.svc, DNS:istio-sidecar-injector.istio-system
+    {{< /text >}}
+
+## `istioctl` enables webhook configurations
+
+1.  Generate `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` by running the following
+command:
+
+    {{< text bash >}}
+    $ helm template \
+        --name=istio \
+        --namespace=istio-system \
+        --values install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml \
```

Comment:
Should these values be customized? In which cases? How?

Reply:
Done: explanations added.